src/current/v25.4/logging-overview.md

```diff
@@ -24,37 +24,61 @@ All [`cockroach` commands]({% link {{ page.version.version }}/cockroach-commands
-| [`SQL_PERF`]({% link {{ page.version.version }}/logging.md %}#sql_perf)                 | SQL executions that impact performance, such as slow queries (when enabled via the `sql.log.slow_query.latency_threshold` and/or `sql.log.slow_query.experimental_full_table_scans.enabled` [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %})).                                                                                    |
-| [`TELEMETRY`]({% link {{ page.version.version }}/logging.md %}#telemetry)               | Telemetry events for internal usage, except for [`sampled_query` events]({% link {{ page.version.version }}/eventlog.md %}#sampled_query) which may be [logged externally to Datadog]({% link {{ page.version.version }}/log-sql-activity-to-datadog.md %}). |
-| [`KV_DISTRIBUTION`]({% link {{ page.version.version }}/logging.md %}#kv_distribution)   | Data distribution events, such as moving [replicas]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-replica) between [stores]({% link {{ page.version.version }}/cockroach-start.md %}#store) in a cluster, adding replicas to [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#range), and removing replicas from ranges. |
 
 Logging channels are analogous to [logging facilities in Syslog](https://wikipedia.org/wiki/Syslog) or [logging services in Datadog](https://docs.datadoghq.com/logs/log_collection/?tab=http#reserved-attributes). For more details on the contents of each logging channel, see the [Logging reference]({% link {{ page.version.version }}/logging.md %}#logging-channels).
 
+### `log.channel_compatibility_mode.enabled`
+
+{% include_cached new-in.html version="v25.4" %} The `log.channel_compatibility_mode.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) when set to:
+
+- `true` (default): Logs are sent to the expected [logging channels](#logging-channels). Specifically:
+
+  - [Changefeed telemetry events]({% link {{ page.version.version }}/eventlog.md %}#changefeed-telemetry-events) are sent to the [`TELEMETRY`](#telemetry) channel.
+  - [SQL Slow Query Log]({% link {{ page.version.version }}/eventlog.md %}#sql-slow-query-log) and [SQL Slow Query Log (Internal)]({% link {{ page.version.version }}/eventlog.md %}#sql-slow-query-log-internal) events are sent to the [`SQL_PERF`](#sql_perf) and [`SQL_INTERNAL_PERF`]({% link {{ page.version.version }}/logging.md %}#sql_internal_perf) channels, respectively.
+  - [`sampled_query`]({% link {{ page.version.version }}/eventlog.md %}#sampled_query) and [`sampled_transaction`]({% link {{ page.version.version }}/eventlog.md %}#sampled_transaction) events are sent to the [`TELEMETRY`](#telemetry) channel.
+
+- `false`: Logs are sent to different logging channels as part of the logging channel consolidation effort. Specifically:
```

Comment:
Logs are sent to the updated logging channels.

Remove "logging channel consolidation effort" users don't know about this